tests/e2e/conftest.py

```diff
@@ -1,18 +1,30 @@
 import json
 from pathlib import Path
-from time import sleep
+from time import sleep, time
 
 import pytest
 
 import requests
 from fhirpy import SyncFHIRClient
 from tenacity import retry, retry_if_exception_type, stop_after_attempt, wait_fixed
+from tests.conftest import load_mapping
 
 from . import settings
 
 DATA_FIXTURES_DIR = Path(__file__).resolve().parent.parent / "fixtures"
 
 
+def destroy_source(source):
```

Comment:
we could just pass a name here and rename this to something like destroy_source_by_name since that's what this function is specificaly doing ? I don't know if it would just work to set autouse=True, no need to call it in uploaded mapping then